cmd/system-probe/probe.go

```diff
@@ -161,14 +161,20 @@ func getClientID(req *http.Request) string {
 }
 
 func writeConnections(w http.ResponseWriter, cs *ebpf.Connections) {
-	buf, err := easyjson.Marshal(cs)
-	if err != nil {
+	jw := &jwriter.Writer{}
+	cs.MarshalEasyJSON(jw)
+	if err := jw.Error; err != nil {
 		log.Errorf("unable to marshall connections into JSON: %s", err)
 		w.WriteHeader(500)
 		return
 	}
-	w.Write(buf)
-	log.Tracef("/connections: %d connections, %d bytes", len(cs.Conns), len(buf))
+	bytes, err := jw.DumpTo(w)
```

Comment:
nit, the naming here makes me think its the actual bytes returned but in reality is a count.

Reply:
Agreed, I changed it to `bytesWritten`